Makefile

```diff
@@ -573,9 +574,13 @@ ISTIO_SIDECAR_SRC=tools/deb/istio-start.sh \
                   security/tools/deb/istio-auth-node-agent.service \
                   tools/deb/sidecar.env tools/deb/envoy.json
 
+# Note: adding --deb-systemd ${GO_TOP}/src/istio.io/istio/tools/deb/istio.service will result in
+# a /etc/systemd/system/multi-user.target.wants/istio.service and auto-start. Currently not used
+# since we need configuration.
 # --iteration 1 adds a "-1" suffix to the version that didn't exist before
 ${OUT}/istio-sidecar.deb: ${ISTIO_BIN}/envoy ${ISTIO_BIN}/pilot-agent ${ISTIO_BIN}/node_agent ${ISTIO_SIDECAR_SRC}
```

Comment:
need to add dependencies for ${ISTIO_BIN}/istioctl ${ISTIO_BIN}/mixs ${ISTIO_BIN}/pilot_discovery

Reply:
Done